frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -50,14 +56,23 @@ describe('Kubevirt create VM using wizard', () => {
   };
 
   beforeAll(async () => {
-    createResources([multusNAD, testDataVolume]);
+    createResources([testDataVolume]);
   });
 
   afterAll(async () => {
-    deleteResources([multusNAD, testDataVolume]);
+    deleteResources([testDataVolume]);
+  });
+
+  beforeEach(async () => {
+    createResources([multusNAD]);
   });
 
   afterEach(() => {
+    try {
+      deleteResources([multusNAD]);
+    } catch (e) {
```

Comment:
what exception are we getting? I though ignoreNotFound should work here

Reply:
it doesnt work, I got an error:
` Error: Command failed: kubectl delete --ignore-not-found=0 -n test-uaxpn --cascade net-attach-def multus-test-uaxpn
   Error from server (NotFound): network-attachment-definitions.k8s.cni.cncf.io "multus-test-uaxpn" not found`

Reply:
can you please debug this part? Wrong command is constucted here:

it should be `--ignore-not-found=true`

Reply:
It get `--ignore-not-found=0` because `resources.forEach` send the index as the the 2nd parameter to `deleteResource`, there is an improvement for `deleteResources` in #6967 to fix this.

Reply:
this is what it needed. all fixed.